README.md

```diff
@@ -640,13 +648,19 @@ To use CMake with C++ modules, you must first enable its experimental support, s
 ```cmake
 # enable C++ module support
 cmake_minimum_required( VERSION 3.25 )
-# test if CMake version is ≥ 3.26
-if ( ${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.26 )
-  # CMake 3.26; need to handle future versions here
-  set( CMAKE_EXPERIMENTAL_CXX_MODULE_CMAKE_API 2182bf5c-ef0d-489a-91da-49dbc3090d2a )
+if ( ${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.28 )
+	message(FATAL_ERROR "VULKAN_HPP_ENABLE_EXPERIMENTAL_CPP20_MODULES is currently not supported for CMake version ${CMAKE_VERSION}!"
+	" To add support inform yourself about the state of the feature at https://github.com/Kitware/CMake/blob/master/Help/dev/experimental.rst"
+	" and add the corresponding value of CMAKE_EXPERIMENTAL_CXX_MODULE_CMAKE_API to Vulkan-Hpp's CMakeLists.txt")
+elseif ( ${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.27 )
+	# CMake 3.27/3.27.1

```

Comment:
maybe replace this with 3.27.X or just 3.27? Since this commit 3.27.3 and 3.27.4 were released with the same value for the cxx module API